src/amo/components/Routes/index.js

```diff
@@ -162,6 +163,12 @@ const Routes = ({ _config = config }: Props = {}): React.Node => (
       component={TagPage}
     />
 
+    <Route
+      exact
+      path="/:lang/:application(firefox|android)/feedback/addon/:addonIdentifier/"
+      component={Feedback}
+    />
```

Comment:
Should we put the route behind the config flag?

Reply:
If that's a thing, rather than doing it in the page, sure!

Reply:
I think we could do:

```js
{config.get("...)" && (
  <Route />
)}
```